src/components/ATATFileUpload.vue

```diff
@@ -1,77 +1,152 @@
 <template>
-  <v-sheet class="atat-file-upload" width="416">
-    <v-card
-      id="file_upload"
-      @drop.prevent="onDrop($event)"
-      @dragover.prevent="dragover = true"
-      @dragenter.prevent="dragover = true"
-      @dragleave.prevent="dragover = false"
-      elevation="0"
-    >
-      <v-card-text>
-        <v-row class="d-flex flex-column" dense align="center" justify="center">
-          <v-icon :class="[dragover ? 'mt-2, mb-6' : 'mt-5']" size="60">
-            upload_file
-          </v-icon>
-          <span class="lead-paragraph">Drag and Drop</span>
-          <span>your file here or <a href="##">browse to upload</a>.</span>
-        </v-row>
-        <v-virtual-scroll
-          v-if="uploadedFiles.length > 0"
-          :items="uploadedFiles"
-          height="150"
-          width="416"
-          item-height="50"
+  <div>
+    <v-flex>
+      <label
+        :id="id + '_text_field_label'"
+        class="form-field-label my-1"
+        :for="id + '_text_field'"
+      >
+        {{ label }}
+        <span class="ml-2 optional" v-show="optional">Optional</span>
+      </label>
+    </v-flex>
+    <v-flex class="my-2">
+      <v-sheet class="atat-file-upload" width="416" v-show="!isFileUploaded">
+        <v-card
+          id="file_upload"
+          @drop.prevent="onDrop($event)"
+          @dragover.prevent="dragover = true"
+          @dragenter.prevent="dragover = true"
+          @dragleave.prevent="dragover = false"
+          elevation="0"
         >
-          <template v-slot:default="{ item }">
-            <v-list-item :key="item.name">
-              <v-list-item-content>
-                <v-list-item-title>
-                  {{ item.name }}
-                  <span class="ml-3 text--secondary">
-                    {{ Math.ceil(item.size / 1024) }} bytes</span
-                  >
-                </v-list-item-title>
-              </v-list-item-content>
-
-              <v-list-item-action>
+          <v-card-text>
+            <v-row
+              class="d-flex flex-column"
+              dense
+              align="center"
+              justify="center"
+            >
+              <v-icon class="mt-1" size="60"> upload_file </v-icon>
+              <div class="lead-paragraph">Drag and Drop</div>
+              <div class="d-flex align-items-center">
+                <div>your file here or</div>
                 <v-btn
-                  class="error"
-                  :outlined="true"
-                  id="remove_file"
-                  @click.stop="removeFile(item.name)"
+                  id="open-file-dialog"
+                  class="link-button body pa-0 ma-0 ml-1"
+                  @click="openFileDialog"
+                  :ripple="false"
```

Comment:
Is disabiling the ripple effect something that we're doing across the project?

Reply:
yep